src/setup.cpp

```diff
@@ -45,32 +40,25 @@ int handle_setup(bool install)
 
 		app_error = apps->AddApplicationManifest(manifest_path.c_str());
 		if (app_error != vr::VRApplicationError_None)
```

Comment:
By convention, `VRApplicationError_None` is zero which is false, so you can simplify (if you want):

```suggestion
		if (app_error)
```


:-)